src/content/en/2020/capabilities.md

```diff
@@ -245,16 +242,16 @@ relatedApps.forEach((app) => {
 
 {{ figure_markup(
   image="get_installed_related_apps.png",
-  alt="Number of top pages using getInstalledRelatedApps()",
-  caption="Number of top pages using getInstalledRelatedApps().",
+  caption="Number of pages using getInstalledRelatedApps().",
   description="Chart of getInstalledRelatedApps() usage, based on the number of pages monitored by HTTPArchive. It compares the usage on mobile and desktop devices. It shows a steady growth for mobile devices, peaking at 363 pages in October 2020 compared to 44 desktop pages.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1774881171&format=interactive",
   sheets_gid="860146688",
   sql_file="get_installed_related_apps_usage.sql"
   )
 }}
 
-Over the course of 2020, the `getInstalledRelatedApps()` API shows a strictly monotone growth on mobile websites. In October, 363 mobile pages tracked by the HTTP Archive made use of this API. On the desktop, the API does not grow quite as fast. This could also be due to Android stores currently providing significantly more apps than the Microsoft Store does for Windows.
+{# TODO(author): Consider rephrasing "strictly monotone growth". #}
+Over the course of 2020, the `getInstalledRelatedApps()` API shows a strictly monotone growth on mobile websites. In October, 363 mobile pages tracked by the HTTP Archive made use of this API. On desktop pages, the API does not grow quite as fast. This could also be due to Android stores currently providing significantly more apps than the Microsoft Store does for Windows.
```

Comment:
```suggestion
Over the course of 2020, the `getInstalledRelatedApps()` API shows a steady growth on mobile websites. In October, 363 mobile pages tracked by the HTTP Archive made use of this API. On desktop pages, the API does not grow quite as fast. This could also be due to Android stores currently providing significantly more apps than the Microsoft Store does for Windows.
```